pkgs/data/documentation/anarchism/default.nix

```
@@ -8,7 +8,7 @@ stdenv.mkDerivation rec {
     domain = "salsa.debian.org";
     owner = "debian";
     repo = pname;
-    rev = "debian%2F${version}"; # %2F = urlquote("/")
+    rev = "debian/${version}"; # %2F = urlquote("/")
```

```suggestion
    rev = "debian/${version}";
```
comment not needed anymore